extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -516,20 +567,20 @@ Type attribute value: `upscore`
 
 Type attribute value: `vponanalytics`
 
-Adds support for Vpon Vpon Analytics. Configuration details can be found at [Vpon Analytics](https://cmp.vpadn.com/dmp/doc/amp_analytics.html).
+Adds support for Vpon Analytics. Configuration details can be found at [Vpon Analytics](https://cmp.vpadn.com/dmp/doc/amp_analytics.html).
 
 ### Webengage
 
-Type attribute `webengage`
+Type attribute value: `webengage`
 
-### Webtrekk
+### Webtrekk (deprecated)
 
-The attribute value ~~`webtrekk`~~ is deprecated (will remove on 31/12/2018) - use `webtrekk_v2` instead
+Type attribute value: `webtrekk`
 
-Adds support for Webtrekk. Configuration details can be found at [supportcenter.webtrekk.com](https://supportcenter.webtrekk.com/en/public/amp-analytics.html).
+> ⚠️ ~~`webtrekk`~~ is deprecated. **Use `webtrekk_v2` instead.**
```

Comment:
Previously:

> The attribute value ~~`webtrekk`~~ is deprecated (will remove on 31/12/2018) - use `webtrekk_v2` instead

@micajuine-ho

Do we have a process for removing vendor types in general? Can we remove this one in particular?

Reply:
I don't believe we do. I would imagine we don't since it would be breaking if someone were to use it and a simple change in the analytics code (or cache) to change `type=webtrekk` to `type=webtrek_v2` might not work b/c inline configs could be incompatible. 

We could do assess how often webtrekk (not v2) is used via cookbook and then evaluate how much of an impact deprecation would have.

Reply:
If we don't have a deprecation process, we don't need to create one. The blurb should be enough.